README.md

```diff
@@ -725,130 +725,10 @@ Step through some actions before Pa11y runs. This example logs into a fictional
 Inject a script before Pa11y runs. This example logs into a fictional site then waits until the account page has loaded before running Pa11y. [See the example](example/before-script/index.js).
 
 
-Common Questions
-----------------
+Common Questions and Troubleshooting
```

Comment:
As this title has changed, the ID of the heading element will have too. This needs to be updated in the table of contents